extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueBindingConstants.java

```diff
@@ -22,7 +22,7 @@
  * @author Jochen Hiller - Added OSRAM Classic A60 RGBW
  * @author Markus Mazurczak - Added OSRAM PAR16 50
  * @author Andre Fuechsel - changed to generic thing types
- *
+ * @author Samuel Leisering & Christoph Weitkamp - Added support for sensor API
```

Comment:
Please add two lines, one for each author (ok to have the same description for both)